elife/config/lib-systemd-system-selenium.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description="Selenium server"
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
```

Comment:
yeah - two places. the equivalent to that link is the systemd docs: https://www.freedesktop.org/software/systemd/man/systemd.service.html and https://www.freedesktop.org/software/systemd/man/systemd.unit.html (for the `[service]` and `[unit]` sections respectively) and the Ubuntu 'systemd for upstart users' page: https://wiki.ubuntu.com/SystemdForUpstartUsers

I started with the ubuntu page, it gives a quick and basic overview with a table of equivalents in there and then drop into the man pages if I need more help. Man pages are a bit terse but there are also plenty of people talking about systemd because almost everything is using it these days